erlef/website
# Include support for ecto_sql & phoenix_ecto

* Adds ecto_sql, phoenix_ecto, & postgrex dependencies
* Setups up configs & Repo
* Adds uuid-ossp extension to postgres with migration
* Updates README for database setup steps

Requirements: Have a local copy of postgres running

QA INFO: 
Download and run `mix do ecto.create, ecto.migrate`
Verify the database has been created locally
Verify the extension has been added be running the following from 

```
$ pgsql erlef_website_dev
erlef_website_dev=# select uuid_generate_v4();
           uuid_generate_v4
--------------------------------------
 38852d8a-3bbe-40c1-865e-7fa6a6065418
```

The result will be different but look should be the same :)
